doctypes/tile.md

```diff
@@ -0,0 +1,135 @@
+
+```JSON
+{
+  "doctype": "tile",
+  "owners": ["did:3:<CID>"],
+  "content": ...
+}
+```
+
+## Signed record
+
+Signed records in 3ID are stored in IPFS using the `dag-jose` or `dag-cose` formats. These formats allow for IPLD objects to be stored along with a standardized signature format. They also allow DAG objects to be encrypted. The update to the document is encoded using [json-patch](https://github.com/Starcounter-Jack/JSON-Patch).
+
+```JSON
+{
+  "next": <CID-previous-record>,
+  "content": <JSON-patch-object>
+}
+```
+
+## Update rules
+
+An update to a tile can change anything in the `content` of the tile document. If there are multiple owners only the first *signed record* needs to contain the json-patch that changes the document. The other owners can simply sign a record that only contains the `next` pointer to the previous signed record.
+
+## Examples
+
+Below there are three examples of possible tiles, Collection, Service, and Privacy tiles. These tiles in particular are useful when specifying a user cenrtic data storage system.
+
+### Collection tile
+
+The **Collection tile** represents a *collection* of data, or rather a *collection* of databases that has specific schemas. This tile also contain tags which describe what type of data is being stored within the collection.
+
+**Example:**
+
+```JSON
+{
+  "doctype": "tile",
+  "owners": ["did:3:bafyappDeveloper..."],
+  "content": {
+    "databases": [{
+      "type": "orbitdb",
+      "store": "kvstore",
+      "access": "legacy-ipfs-3box",
+      "schema": ...
+    }, {
+      "type": "orbitdb",
+      "store": "feedstore",
+      "access": "thread-access"
+      "schema": ...
+    }],
+    "tags": ["profile"]
+  }
+}
+```
+
+### Service tile
+
+A **Service tile** is used to describe a simple service, usually in the form of a function that takes an input and produces an output. But could also describe more complex services such as database hosting etc.
+
+**Example:**
+
+This is an example of how the service tile for the Ceramic anchoring service could look like.
+
+```JSON
+{
+  "doctype": "tile",
+  "owners": ["did:3:bafyserviceProvider..."],
+  "content": {
+    "serviceEndpoint": "http://myservice.com/api/v0/",
+    "serviceFunctions": [{
+      "name": "requestTimestamp",
+      "parameterSchema": ...,
+      "resultSchema": ...
+    }, {
+      "name": "resolveTimestamp",
+      "parameterSchema": ...,
+      "resultSchema": ...
+    }]
+  }
+}
+```
+
+### Privacy tile
+
+A **Privacy tile** allows the user to specify their database instances in a specific *collection* and set special access control rights for these databases. In it the *source* of the data, i.e. the **Service tile** that provides a hosting service, is set which contains the information about where data is backed up. A user can of course set multiple *sources* for a given *collection* for redundant backups, or even change where data is being hosted.
```

Comment:
Change "Privacy tile" to: "Privacy Policy tile"
Change "Service tile" to: "Service Policy tile"